packages/block-editor/src/components/block-list/block-mobile-toolbar.native.js

```diff
@@ -45,6 +51,28 @@ const BlockMobileToolbar = ( {
 	</View>
 );
 
+const BlockMobileToolbarItems = ( { onSettingsToggle } ) => {
+	return (
+		<InspectorControls.Slot
```

Comment:
As we [discussed](https://github.com/WordPress/gutenberg/pull/17279#discussion_r319948856) InspectorControls is an independent thing from other toolbar buttons. So it doesn't look right to merge `settings button` with other buttons under **InspectorControls.Slot buttons**. For now we don't have to have a slot for other buttons for simplicity, we can remove BlockMobileToolbarItems altogether and  just add a `ToolbarButton` instead.

Reply:
I see the point please check my comment above under @koke comment